swagger-gen/restapi/embedded_spec.go

```diff
@@ -69,7 +69,7 @@ func init() {
         "description": "Returns a list of all service names associated with span endpoints.\n",
         "responses": {
           "200": {
-            "description": "Succes",
+            "description": "Success",
```

Comment:
This actual error comes from https://github.com/jaegertracing/jaeger-idl/blob/master/swagger/zipkin2-api.yaml#L21